### PR TITLE
nload: add new package

### PR DESCRIPTION
**Pull request by LXGMAX**
Signed-off-by: Jacky Guo <leickwell@hotmail.com>

Maintainer: me
Compile tested: aarch64/mt7622), ramips/mt7628
Run tested: Redmi AX6S, Minew G1-C

Description:
A console application which monitors network
traffic and bandwidth usage in real time.
https://github.com/rolandriegel/nload

### CODE REVIEW

File: admin/nload/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+PKG_NAME:=nload
+PKG_VERSION:=0.7.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.roland-riegel.de/nload/
+PKG_HASH:=c1c051e7155e26243d569be5d99c744d8620e65fa8a7e05efcf84d01d9d469e5
+
+PKG_MAINTAINER:=Jacky Guo <leickwell@Hotmail.com>
```
Comment (PolynomialDivision) [inline admin/nload/Makefile:15]: `Hotmail`? I would assume `hotmail`?

File: admin/nload/Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
```
Comment (PolynomialDivision) [inline admin/nload/Makefile:2]: Please use SPDX-Identifier.

File: admin/nload/Makefile (hunk 1 above)
Comment (LXGMAX) [reply admin/nload/Makefile:15]: It's a bit difficult cause I type "Hotmail" when register GitHub.

File: admin/nload/Makefile (hunk 2 above)
Comment (LXGMAX) [reply admin/nload/Makefile]: > Please use SPDX-Identifier.

Done.

File: admin/nload/Makefile (hunk 1 above)
Comment (LXGMAX) [reply admin/nload/Makefile]: > `Hotmail`? I would assume `hotmail`?

OK, I'm fixed.

File: admin/nload/Makefile [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# SPDX-Identifier-License: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nload
+PKG_VERSION:=0.7.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.roland-riegel.de/nload/
```
Comment (BKPepe) [inline admin/nload/Makefile:10]: I'm thinking, don't we want to use secure way to download source files by using [nload's GitHub repository](https://github.com/rolandriegel/nload)?

File: admin/nload/Makefile [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/nload
+  SECTION:=admin
+  CATEGORY:=Administration
+  TITLE:=Real time network traffic monitor for the text console
```
Comment (BKPepe) [inline admin/nload/Makefile:26]: Isn't this too long for ``make menuconfig``?

File: admin/nload/Makefile [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  DEPENDS:=+libncurses +libstdcpp
+  URL:=https://github.com/rolandriegel/nload
+endef
+
+define Package/nload/description
+  monitors network traffic and bandwidth usage
```
Comment (BKPepe) [inline admin/nload/Makefile:32]:
```suggestion
  Monitors network traffic and bandwidth usage.
```

**Review by BKPepe — COMMENTED** [review]
What would be handy is to add ``test.sh`` to have it run tested in CI.

File: admin/nload/Makefile (hunk 3 above)
Comment (LXGMAX) [reply admin/nload/Makefile]: Sure, I update source url to GitHub.

File: admin/nload/Makefile (hunk 5 above)
Comment (LXGMAX) [reply admin/nload/Makefile]: Fixed.

**Review by BKPepe — APPROVED** [review]

**Review by PolynomialDivision — APPROVED** [review]

**Pull request merged**